lua_modules/http/httpserver.lua

```diff
@@ -23,54 +23,49 @@ do
   -- response methods
   ------------------------------------------------------------------------------
   local make_res = function(csend, cfini)
-   local send = function(self, data, status)
-    -- TODO: req.send should take care of response headers!
-    if self.send_header then
-      csend("HTTP/1.1 ")
-      csend(tostring(status or 200))
-      -- TODO: real HTTP status code/name table
-      csend(" OK\r\n")
-      -- we use chunked transfer encoding, to not deal with Content-Length:
-      --   response header
-      self:send_header("Transfer-Encoding", "chunked")
-      -- TODO: send standard response headers, such as Server:, Date:
+   -- send("hello world\r\n") will send the data
+    -- send() will close connection
```

Comment:
Try something like "`send(data)` sends the string `data` to the client as a HTTP chunk.  If `data` is `nil`, the stream will be terminated and the connection closed."

Reply:
the new `send` already look like this. when use `send()` will close the connection.

Reply:
Sorry to be unclear; I was just meaning to replace the comment with something a little more descriptive and less example-centric.

Reply:
-- send(nil) ?